### PR TITLE
In REST API for usage: LBaaS is Neutron

### PR DESCRIPTION
**Pull request by bpetermannS11**
Clarify that the network.lb usage refers to Neutron LBaaS

### CODE REVIEW

File: user/pages/04.Reference/10.get-quota-info/docs.en.md [hunk 1]
```diff
@@ -116,11 +116,11 @@ compute.instances | Number of virtual machines (servers, instances) |
 compute.ram_mb | RAM for virtual machines in MiB |
 dns.zones | Number of DNS zones |
 network.floatingips | Number of floating IP addresses |
-network.lb_healthmonitors | Number of Neutron LBaaSv2 health monitors |
-network.lb_listeners | Number of Neutron LBaaSv2 listeners |
-network.lb_members | Number of Neutron LBaaSv2 pool members |
-network.lb_pools | Number of Neutron LBaaSv2 pools |
-network.loadbalancers | Number of Neutron LBaaSv2 load balancers |
+network.lb_healthmonitors | Number of Neutron LBaaS health monitors |
```
Comment (matthiasbernhardt) [inline user/pages/04.Reference/10.get-quota-info/docs.en.md:119]: Why did you remove the "v2"? There were to my knowledge two kinds of Neutron LBaaS and we never supported the Neutron LBaaS (v1).
In OpenStack Docs I consistently read "LBaaS v2" and we should probably insert the space but not omit the "v2" entirely.
https://wiki.openstack.org/wiki/Neutron/LBaaS/Deprecation mentions
Neutron-lbaas vs. LBaaS v2 API vs. Octavia vs. Octavia v2 API
For our purpose I'd suggest to distinguish between "Neutron LBaaS v2" and "Octavia LBaaS".

File: user/pages/04.Reference/10.get-quota-info/docs.en.md [hunk 2]
```diff
@@ -156,13 +156,15 @@ avoided.
...
-- network.lb
-- network.vpn
-- s3
-- volume
+Filter name | Details |
+------------|---------|
+compute     | Compute resources |
+dns         | DNS zones |
+network     | Network resources (excl. LBaaS and VPNaaS) |
+network.lb  | Neutron LBaaS resources |
```
Comment (matthiasbernhardt) [inline user/pages/04.Reference/10.get-quota-info/docs.en.md:164]: Neutron LBaaS v2, see above

File: user/pages/04.Reference/10.get-quota-info/docs.en.md [hunk 3]
```diff
@@ -240,11 +242,11 @@ compute.instances | Total number of virtual machines |
 compute.ram_mb | Total RAM of virtual machines in MiB |
 dns.zones | Number of DNS zones |
 network.floatingips | Number of floating IP addresses |
-network.lb_healthmonitors | Number of LBaaS health monitors |
-network.lb_listeners | Number of LBaaS listeners |
-network.lb_members | Number of LBaaS pool members |
-network.lb_pools | Number of LBaaS pools |
-network.loadbalancers | Number of LBaaS load balancers |
+network.lb_healthmonitors | Number of Neutron LBaaS health monitors |
```
Comment (matthiasbernhardt) [inline user/pages/04.Reference/10.get-quota-info/docs.en.md:245]: Neutron LBaaS v2, see above

**Review by matthiasbernhardt — CHANGES_REQUESTED** [review]
For our purpose I'd suggest to distinguish between "Neutron LBaaS v2" and "Octavia LBaaS".

File: user/pages/04.Reference/10.get-quota-info/docs.en.md (hunk 1 above)
Comment (bpetermannS11) [reply user/pages/04.Reference/10.get-quota-info/docs.en.md]: added v2 again

File: user/pages/04.Reference/10.get-quota-info/docs.en.md (hunk 2 above)
Comment (bpetermannS11) [reply user/pages/04.Reference/10.get-quota-info/docs.en.md]: done

**Review by matthiasbernhardt — APPROVED** [review]
lgtm

**Pull request merged**